Source: Locke 'Wizzleby' Shinseiko , Gentoo bug 308651
Upstream: Not sure
Reason: Fix compilation with libpng14.
diff -pur extremetuxracer-0.4.orig//src/ppgltk/images/png_reader.cpp extremetuxracer-0.4/src/ppgltk/images/png_reader.cpp
--- extremetuxracer-0.4.orig//src/ppgltk/images/png_reader.cpp	2010-03-09 09:35:16.899211768 -0500
+++ extremetuxracer-0.4/src/ppgltk/images/png_reader.cpp	2010-03-09 09:46:14.087212197 -0500
@@ -77,7 +77,7 @@ ReaderPNG::ReaderPNG(const char *fileNam
 	
 	
 	png_get_IHDR(png_ptr, info_ptr, &width, &height,
-       &bit_depth, &color_type, &interlace_type,int_p_NULL, int_p_NULL);
+       &bit_depth, &color_type, &interlace_type,(int *) NULL, (int *) NULL);
 	
 	if(bit_depth == 16)
         png_set_strip_16(png_ptr);
@@ -88,7 +88,7 @@ ReaderPNG::ReaderPNG(const char *fileNam
 		png_set_expand(png_ptr);
 		png_read_update_info(png_ptr, info_ptr);
 		png_get_IHDR(png_ptr, info_ptr, &width, &height,
-			   &bit_depth, &color_type, &interlace_type,int_p_NULL, int_p_NULL);
+			   &bit_depth, &color_type, &interlace_type,(int *) NULL, (int *) NULL);
 	}
 	
 	if( color_type == PNG_COLOR_TYPE_GRAY ||
@@ -96,7 +96,7 @@ ReaderPNG::ReaderPNG(const char *fileNam
 		png_set_gray_to_rgb(png_ptr);
 		png_read_update_info(png_ptr, info_ptr);
 		png_get_IHDR(png_ptr, info_ptr, &width, &height,
-			   &bit_depth, &color_type, &interlace_type,int_p_NULL, int_p_NULL);
+			   &bit_depth, &color_type, &interlace_type,(int *) NULL, (int *) NULL);
 	}
 		
 	this->width=width;
@@ -114,7 +114,7 @@ ReaderPNG::ReaderPNG(const char *fileNam
 	}
 	
 	png_read_end(png_ptr, info_ptr);
-	png_destroy_read_struct(&png_ptr, &info_ptr, png_infopp_NULL);
+	png_destroy_read_struct(&png_ptr, &info_ptr, NULL);
 	fclose(fp);
 }
 
